app/src/main/java/com/infomaniak/mail/MatomoMail.kt

```diff
@@ -248,6 +249,10 @@ object MatomoMail : MatomoCore {
         trackEvent("inAppUpdate", name)
     }
 
+    fun View.trackCalendarEventEvent(name: String, value: Float? = null) {
```

Comment:
For the value, directly pass a boolean instead of a float, then use the extension toMailActionValue on it, to abstract the cast

Reply:
I passed a Boolean instead of a Float and made the `toFloat()` call inside the function rather than outside of it